### PR TITLE
📝(content) Post: ADB over Wi-Fi and Ethernet

### PR DESCRIPTION
**Pull request by jkcinouye**
Content added to posts:
ADB over Wi-Fi and Ethernet

posts/adb-over-wifi.md

### CODE REVIEW

**Review by Snugug — CHANGES_REQUESTED** [review]
Two items:
* This is Android documentation, should live in `android` instead of `posts`
* Images need to have their URLs updated

File: site/en/android/adb-over-wifi.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  - android
+---
+
+Android developers need to test their apps on real devices. [Android Debug Bridge (ADB)](https://developer.android.com/tools/adb) is the tool of choice for pushing apps and debugging. There are three main ways ADB can be used:
+
+1.  ADB over USB
+1.  ADB over Wi-Fi
+1.  ADB over Ethernet
+
+For most phones and tablets, you can connect an Android device to your development machine using a USB-C cable and ADB will "just work." As of June 2023, most Chromebooks do not support ADB over USB, leaving the two other methods: Wi-Fi and Ethernet.
```
Comment (Snugug) [inline site/en/android/adb-over-wifi.md:15]:
```suggestion
For most phones and tablets, you can connect an Android device to your development machine using a USB-C cable and ADB will "just work." As of August 2023, most Chromebooks do not support ADB over USB, leaving the two other methods: Wi-Fi and Ethernet.
```

File: site/en/android/adb-over-wifi.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+- **Ethernet cable:** A physical cable to run between your devices. Ethernet types Cat 5/5e/6/6a are all fine.
+- **Ethernet port:** An Ethernet port for both your test device and your development machine.
+
+!!! aside.message--note
+**Note:** If your device does not have an Ethernet port, you can instead use an Ethernet dongle that connects to your device via USB and provides Ethernet capability.
+!!!
+
+![A USB-A Ethernet dongle.](ix://android/adb/ethernet.jpg)
+_A USB-A Ethernet dongle._
```
Comment (Snugug) [inline site/en/android/adb-over-wifi.md:54]:
```suggestion
#[A USB-A Ethernet dongle.](ix://android/adb/ethernet.jpg)
```

We actually have a dedicated figure element specifically for this kind of setup. Replacing the `!` with a `#` should do it.

File: site/en/android/adb-over-wifi.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+![A complete ADB over Ethernet setup including USB-A Ethernet dongle, CAT6 Ethernet cable, USB-A Ethernet dongle, and USB-A to USB-C adapter.](ix://android/adb/double-ethernet.jpg)
+_A complete ADB over Ethernet setup, including: USB-A Ethernet dongle, CAT6 Ethernet cable, USB-A Ethernet dongle, and USB-A to USB-C adapter._
```
Comment (Snugug) [inline site/en/android/adb-over-wifi.md:57]:
```suggestion
#[A complete ADB over Ethernet setup including USB-A Ethernet dongle, CAT6 Ethernet cable, USB-A Ethernet dongle, and USB-A to USB-C adapter.](ix://android/adb/double-ethernet.jpg)
```

Same here.

**Review by jkcinouye — COMMENTED** [review]
Images added!
URL changed!

File: site/en/android/adb-over-wifi.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+Android developers need to test their apps on real devices. [Android Debug Bridge (ADB)](https://developer.android.com/tools/adb) is the tool of choice for pushing apps and debugging. There are three main ways ADB can be used:
+
+1.  ADB over USB
+1.  ADB over Wi-Fi
+1.  ADB over Ethernet
+
+For most phones and tablets, you can connect an Android device to your development machine using a USB-C cable and ADB will "just work." As of June 2023, most Chromebooks do not support ADB over USB, leaving the two other methods: Wi-Fi and Ethernet.
+
+Note: You can also use [ADB over Bluetooth](https://developer.android.com/training/wearables/get-started/debugging) for WearOS devices
```
Comment (Snugug) [inline site/en/android/adb-over-wifi.md:17]:
```suggestion
!!! aside.message--note
**Note:** You can also use [ADB over Bluetooth](https://developer.android.com/training/wearables/get-started/debugging) for WearOS devices
!!!
```

Making this a note popout

**Comment by fuentespatrick** [comment]
The other ADB content lives here: https://chromeos.dev/en/android-environment/deploying-apps 
Do we want to keep them together and/or reference them from each other?

File: site/en/android/adb-over-wifi.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+# ADB over Wi-Fi
```
Comment (Snugug) [inline site/en/android/adb-over-wifi.md:19]:
```suggestion
## ADB over Wi-Fi
```

File: site/en/android/adb-over-wifi.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+    1.  On phones/tablets, go to Android settings, information, and scroll down until you see your **IP address**.
+    1.  On ChromeOS, click the clock at the bottom right, click the drop-down under the network icon, and then click on the network you are currently connected to. Under the **Network** heading you will see the IP address assigned to your Chromebook.
+1.  On your development machine, type `adb connect IP_ADDRESS_FROM_STEP4`, e.g., `adb connect 192.168.1.4`. You can also manually specify the port like `adb connect 192.168.1.4:5555`.
+1.  Use ADB or Android Studio as usual. You can verify the connection by running `adb devices`.
+
+!!! aside.message--tip
+**Tip:** For more information, see [Connect to a device over Wi-Fi](https://developer.android.com/tools/adb#wireless).
+!!!
+
+# ADB over Ethernet
```
Comment (Snugug) [inline site/en/android/adb-over-wifi.md:38]:
```suggestion
## ADB over Ethernet
```

File: site/en/android/adb-over-wifi.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+In some cases, it may not be possible to use a Wi-Fi network due to firewall restrictions, or a wired solution may simply be preferable. In these cases, you can connect over an Ethernet cable. The steps are similar to ADB over Wi-Fi but using an Ethernet cable.
+
+## Prerequisites
```
Comment (Snugug) [inline site/en/android/adb-over-wifi.md:42]:
```suggestion
### Prerequisites
```

File: site/en/android/adb-over-wifi.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+**Note:** If your device does not have an Ethernet port, you can instead use an Ethernet dongle that connects to your device via USB and provides Ethernet capability.
+!!!
+
+![A USB-A Ethernet dongle.](ix://android/adb/ethernet.jpg)
+_A USB-A Ethernet dongle._
+
+![A complete ADB over Ethernet setup including USB-A Ethernet dongle, CAT6 Ethernet cable, USB-A Ethernet dongle, and USB-A to USB-C adapter.](ix://android/adb/double-ethernet.jpg)
+_A complete ADB over Ethernet setup, including: USB-A Ethernet dongle, CAT6 Ethernet cable, USB-A Ethernet dongle, and USB-A to USB-C adapter._
+
+## Connecting ADB over Ethernet
```
Comment (Snugug) [inline site/en/android/adb-over-wifi.md:59]:
```suggestion
### Connecting ADB over Ethernet
```

File: site/en/android/adb-over-wifi.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+Once you have physically connected your devices with an Ethernet cable, you can enable ADB over the connection.
+
+!!! aside.message--note
+**Note:** These instructions assume your local network is running on the `192.168.1.x` address space.
+!!!
+
+!!! aside.message--alert
```
Comment (Snugug) [inline site/en/android/adb-over-wifi.md:67]:
```suggestion
!!! aside.message--note
```

We don't have an "alert" type so this is rendering as a standard message, which isn't the vibe we're going for.

**Review by Snugug — CHANGES_REQUESTED** [review]
Sorry, found a few more things when looking at the updated items actually rendered. These should all be click to fix.

**Review by Snugug — DISMISSED** [review]

**Review by Snugug — APPROVED** [review]

**Pull request merged**